repo/packages/H/history/0/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {},
```

Comment:
Currently http://leader.mesos:5050 is used. I make this a configuration parameter which defaults to this value.